tutorials/tools/format-css.md

```diff
@@ -0,0 +1,32 @@
+# Format Your CSS
+
+Input your css into the text field and it will be formatted.
+An example:
+`.nav_bar {color: red; background-color: black; font-size: small} .test {color: blue; background-color: yellow} .footNotes { color: red; background-color: black; transform: translate(120px, 50%)}`
+
+<textarea id="input" cols=50 rows=20></textarea>
```

Comment:
Let's use CodeMirror for this (https://thecodebarbarian.com/building-a-code-editor-with-codemirror.html) so we can get some syntax highlighting, line numbers, etc.